### DISCUSSION TITLE
How to execute query with an empty line inside

### DISCUSSION THREAD

**Discussion by 1248593560** [dbeaver/dbeaver#19927] [Q&A]
### Description

直接执行sql的话会报错，表或者试图不存在
![image](https://github.com/dbeaver/dbeaver/assets/32057857/946b3b91-9d0a-4261-9c34-eec68e517695)
全部选中后即可执行成功
![image](https://github.com/dbeaver/dbeaver/assets/32057857/001eda54-c0a5-4131-8da9-1bbb582693aa)

### DBeaver Version

社区版23.0.4

### Operating System

windows 10

**Comment by LonwoLonwo** [comment]
Hello @1248593560 

I see an empty string on your screen. Please remove it and try again.

![2023-05-10 11_27_37-oracle 数据库下使用WITH AS 语句 进行查询，如果不进行全选执行会报找不到表异常 · Issue #19925 · dbeaver_dbeaver ](https://github.com/dbeaver/dbeaver/assets/45152336/fcf9654d-9d6e-4ddc-b0f0-172f5bc11c93)


By default, DBeaver recognizes empty strings as delimiters. You can change this behavior in Preferences:

![2023-05-10 11_29_40-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/04f2e799-b748-41cd-9051-10623dcc1b2a)
